qa-tests-backend/src/main/groovy/services/NotifierService.groovy

```diff
@@ -10,7 +10,9 @@ import util.Env
 
 @Slf4j
 class NotifierService extends BaseService {
-    private static final PAGERDUTY_API_KEY = "fix-me-ROX-7589-and-this-should-be-secret"
+    // FIXME(ROX-7589): this should be secret
+    // private static final PAGERDUTY_API_KEY = Env.mustGetPagerdutyApiKey()
+    private static final String PAGERDUTY_API_KEY = null
```

Comment:
It looks that GitGuardian still fires an error. Maybe we could try with a comment to disable GitGaurdian for this variable.
```
// ggignore
```
More info: https://docs.gitguardian.com/internal-repositories-monitoring/gg_shield/configuration#ignore-list

```suggestion
    private static final String PAGERDUTY_API_KEY = null // ggignore
```

Reply:
It fires error on previous value https://github.com/stackrox/stackrox/commit/a339157a430bcdff685de22147109adff289471d#diff-57f5f39a218a6c9bae5664f4295768277598091541a6f69ec872398afb337622L13 not new (null)

Reply:
I've checked how GitGuardian works. It's a bit tricky. I think we can go forward and merge this because we have at least a comment that a secret should be adequately provided in the future.